test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FileHash.Tests.ps1

```diff
@@ -3,8 +3,10 @@
 Describe "Get-FileHash" -Tags "CI" {
 
     BeforeAll {
-        $testDocument = Join-Path -Path $PSScriptRoot -ChildPath assets testablescript.ps1
```

Comment:
I don't see that script being used anywhere else, so you should delete that script if it's no longer needed.

Reply:
Deleted in latest commit